x-pack/plugins/cases/server/client/cases/client.ts

```diff
@@ -82,6 +85,10 @@ export interface CasesSubClient {
    * Retrieves the case IDs given a single alert ID
    */
   getCaseIDsByAlertID(params: CaseIDsByAlertIDParams): Promise<string[]>;
+  /**
+   * Retrieves all alerts attach to a case given a single case ID
+   */
+  getAllAlertsAttachToCase(params: GetAllAlertsAttachToCase): Promise<AlertResponse>;
```

Comment:
Just a reminder that we'll want to move `getAllAlertsAttachToCase` to the `attachments` directory.